packages/ag-grid-theme/css/uitk-ag-theme.scss

```diff
@@ -0,0 +1,214 @@
+@import "./uitk-ag-theme-mixin";
+
+.uitk-light .ag-theme-uitk-high,
+.uitk-light .ag-theme-uitk-medium,
+.uitk-light .ag-theme-uitk-low,
+.uitk-light .ag-theme-uitk-touch {
+  --uitk-ag-foreground-color: #2a2c2f;
+  --uitk-ag-disabled-foreground-color: #161616;
```

Comment:
we tend to leave out -color on the end of background and foreground values, and also any states should go on the end of the token (-disabled suffix) - can see figma or storybook docs for more info on ordering

also attributes tend to go in camel case, so e.g. --uitk-ag-row-borderColor

Reply:
Thank you for reviewing this PR. I've renamed vars, please have a look.